libguides/3_column_side_nav_without_profile.html

```diff
@@ -0,0 +1,102 @@
+<!-- SIDE NAV -->
+<body class="s-lg-guide-body">
+{{skip_link}}
+{{ADMIN_CMD_BAR}}
+<!-- BEGIN: Page Header -->
+{{public_header}}
+<!-- END: Page Header -->
+<!-- BEGIN: Guide Header -->
+<div id="s-lg-guide-header" class="container s-lib-header s-lib-side-borders">
+    <nav id="s-lib-bc" aria-label="breadcrumb" role="navigation">
+        {{breadcrumbs}}
+    </nav>
+    <div class="pull-right">
+        <div id="s-lg-guide-header-search">
+            {{guide_search}}        </div>
+        {{ADMIN_MINI_CMD_BAR}}
+    </div>
+    <div id="s-lg-guide-header-info">
+        <h1 id="s-lg-guide-name">{{guide_title}}</h1>
+        <div id="s-lg-guide-desc-container">
+            <span id="s-lg-guide-description">{{guide_description}}</span>
+        </div>
+    </div>
+</div>
+<!-- END: Guide Header -->
+<div id="s-lg-guide-tabs-title-bar" class="container s-lib-side-borders"></div>
+<div id="s-lg-side-nav-content" class="container s-lib-side-borders pad-top-med">
+    <div class="row">
+        <div class="col-md-3 s-lg-tabs-side pad-bottom-med">
+            <div id="s-lg-tabs-container">
+                <div id="s-lg-guide-tabs" role="navigation" aria-label="Guide Pages">
+                    <ul class="nav nav-pills nav-stacked split-button-nav">
+                        {{guide_nav}}
+                    </ul>
+                    {{side_nav_boxes}}
+                </div>
+            </div>
+        </div>
+        <div class="col-md-6">
+            <div class="s-lg-tab-content">
+                <div id="s-lg-guide-main" class="tab-pane active" tabindex="-1">
+                    {{content_col_2}}
+                </div>
+            </div>
+            {{page_prev_next_links}}
+        </div>
+      <div class="col-md-3">
+         {{content_col_3}}
+</div>
+    </div>
+</div>
+
+<style>
+.s-lib-box-std .s-lib-box-title {
+  color: #1a1a1a;
+  background-image: none;
+  background: none;
+  border-color: var(--pu-black);
+  border-width: 2px;
+  font-size: 1.2875rem;
+  font-weight: 500;
+  margin-bottom: 1rem;
+  padding: 0;
+}
+</style>
+
+<!-- BEGIN: Page Footer -->
+<div id="s-lib-footer-public"
+     class="s-lib-footer footer container  s-lib-side-borders">
+    <div id="s-lg-guide-header-meta" class="pad-top-sm pad-left-med clearfix">
+        <ul id="s-lg-guide-header-attributes" class="">
+            <li id="s-lg-guide-header-updated">
+                {{last_updated_display}}</li>
+        </ul>
+        <div id="s-lib-footer-login-link" class="pull-right pad-right-med">
+            {{login_url_display}}</div>
+    </div>
```

Comment:
Same question as with #22: is the commented-out code needed?  If so, can we add a comment for future-us to know what it is about?  Thanks!